docs/getting-started/building-bpfman.md

```diff
@@ -200,6 +200,22 @@ sudo dnf install llvm-devel clang-devel elfutils-libelf-devel
 sudo apt install clang lldb lld libelf-dev gcc-multilib
 ```
 
+### Install libssl Development Library
```

Comment:
We were trying to remove the SSL dependency. Do you know if this was needed for the examples or bpfman?

Reply:
I think I got into that habit with my personal notes. You install once, but revisit the notes multiple times to remember how to do something and the install instructions are in the way. So I add them to the bottom with a link. But for this, it makes sense to not follow that pattern.